pkg/configuration/config.go

```diff
@@ -201,7 +219,7 @@ func NewConfig(portalURL string, opts ...CreateOpt) (*Config, error) {
 
 	if config.useGLToken || config.trf != nil {
 		if err := validateGLConfig(*config); err != nil {
-			return config, fmt.Errorf("configuration error: %v", err)
+			return config, fmt.Errorf("configuration error: %+v", config)
```

Comment:
Did you  mean to remove err? Seems like err msg returned would be good. Can add config output as additional data, but err should be included.

Reply:
Nah, changed for debugging. Fixed it